source/_posts/2023-09-01-php-core-roundup-16.md

```diff
@@ -0,0 +1,97 @@
+
+---
+title: 'PHP Core Roundup #16'
+layout: post
+tags:
+  - roundup
+author:
+  - name: Ayesh Karunaratne
+    url: https://aye.sh
+  
+  - name: Sergey Panteleev
+    url: https://sergeypanteleev.com
+published_at: 01 September 2023
+
+---
+
+Welcome back to [PHP Core Roundup](/blog/tag/roundup/) series! This is post #16, where we highlight and celebrate the improvements made to PHP during the month past by the PHP development team, members of the PHP Foundation, and more.
+
+> The PHP Foundation is a collective of PHP contributors, veterans, and companies that collaborate to ensure the long-term sustainability of the PHP programming language. The foundation currently supports six contributors focused on PHP's maintenance, debt reduction, and feature development. These contributors work closely with others on coding, documentation, and discussions.
+
+> Started over a year ago, the PHP Core Roundup series offers a summary of the latest developments, discussions, and news about PHP Core, contributed by both PHP Foundation members and other participants. This post is the thirteenth in the PHP Core Roundup series.
+
+{% include "newsletter.html" %}
+
+## PHP 8.3 Branching Out
+
+PHP 8.3 has reached its [feature-freeze](/blog/2023/08/01/php-core-roundup-15/#php-8.3-feature-freeze), and [release managers](/blog/2023/05/02/php-core-roundup-12/#php-8.3-release-managers-elected) branched out the PHP-8.3 branch on August 31.
+
+Now that PHP 8.3 is in a separate branch, the `master` branch will be the development source for PHP 8.4. Bug fixes and other improvements will be cherry-picked for PHP 8.3 (and older branches) as appropriate, but new features that are made to the master branch will not be merged to the PHP 8.3 branch.
+
+Tools that build PHP based on the Git branches will also see a new branch, and the builds from the `master` branch will be named “PHP 8.4” for the first time.
+
+## Releases
+
+The PHP development team released three new versions in August 2023:
+
+**[PHP 8.2.9](https://www.php.net/archive/2023.php#2023-08-16-1)**, **[PHP 8.1.22](https://www.php.net/archive/2023.php#2023-08-03-1)**, and **[PHP 8.0.30](https://www.php.net/archive/2023.php#2023-08-04-1)**
```

Comment:
What do you think about today's updates?

```suggestion
The PHP development team released five new versions in August 2023:

**[PHP 8.2.10](TODO)** and **[PHP 8.1.23](TODO)**

These releases include several bug fixes and improvements, notably in areas such as CLI, Date, Core, DOM, FFI, Hash, MySQLnd, Opcache, PCNTL, SPL, and Standard.

**[PHP 8.2.9](https://www.php.net/archive/2023.php#2023-08-16-1)**, **[PHP 8.1.22](https://www.php.net/archive/2023.php#2023-08-03-1)**, and **[PHP 8.0.30](https://www.php.net/archive/2023.php#2023-08-04-1)**
```

Reply:
I'm already seeing the tags and even Windows builds, I'd say these will be announced in the mailing list shortly too, so we can mention them.